awscli/examples/scheduler/create-schedule-group.rst

```diff
@@ -0,0 +1,14 @@
+**To create a schedule group**
+
+The following ``create-schedule-group`` will create a schedule group. ::
+
+	aws scheduler create-schedule-group \
```

Comment:
All your examples are using tabs for indenting. An indent should be 4 spaces, and all your examples need to be updated with this fix.
https://aws.github.io/aws-cli/docs_styleguide.html#general

Reply:
file fixed